### DISCUSSION TITLE
Kata 3 no ejecuta luego de instalar el entorno virtual

### DISCUSSION THREAD

**Discussion by JacomeGerardo** [LaunchX-InnovaccionVirtual/CursoIntroPython#109] [Q&A]
Luego de haber realizado la Kata 2, mi visual studio code me da este problema. Ya intenté instalar lo que el programa me indica pero me da el mismo error, y he buscado posibles soluciones pero aún no consigo solucionarlo.

¿Alguien aquí puede ayudarme? Mil gracias, soy nuevo en esto :(
![Captura de pantalla (509)](https://user-images.githubusercontent.com/99065797/153533658-a5b026c3-47ea-4087-87fb-fc68fcd48d82.png)

**Comment by BarajasAngel** [comment]
Prueba a utilizar este `python3.10 -m pip install ipykernel` si no llega a funcionar tambien puedes probar sin el 3.10 `python -m pip install ipykernel` esto directamente desde el cmd

**Reply by JacomeGerardo** [reply]
Ahora me da este problema
![image](https://user-images.githubusercontent.com/99065797/153536128-c60a9b8c-cc37-4371-8015-65dfd0baf5e9.png)

**Reply by AlexString** [reply]
Hola compañero, por favor toma estas consideraciones cuando busques ayuda con un error para que alguien pueda ayudarte más eficientemente:

- Leer los mensajes de error, los programadores debemos leer los mensajes de error y googlearlos, (ese es nuestro primer acercamiento para solucionar un problema)
- Mencionar en qué sistema operativo estás trabajando (Win8, 10, 11, Linux, etc)

Si te fijas en las letras en rojo menciona esto:

> This system does not have windows long path support enabled, you can find information on how to enable this at *link*

Por favor intenta obtener más información en el link que menciona el mensaje de error y si no intenta googlear "Enable long path support <tu sistema operativo>"

Te dejo el link que sale en tu screenshot.

https://pip.pypa.io/warnings/enable-long-paths

También en una Googleada rápida que me tomé encontre [esto.](https://support.checkmarx.com/CheckmarxCustomerServiceCommunity/s/article/How-to-Enable-Support-for-Long-Path-in-Windows-10-and-Windows-Server-2016)

¡Saludos y no te desanimes! ¡Es el pan de cada día resolver errores cuando uno programa!

**Comment by doguedogue** [comment]
Ntp siempre se puede instalar/desinstalar algo que de problemas, vuelve  a la kata 0 y te debería funcionar todo.
Avísanos si lo resolviste.
Me gusta tu carpeta almacén 😂
